### PR TITLE
acpic: init at 1.0.0

### PR DESCRIPTION
**Pull request by aacebedo**
## Description of changes

acpic: init at 1.0.0

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/acpic/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "acpic";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit version;
+    inherit pname;
+    sha256 = "sha256-vQ9VxCNbOmqHIY3e1wq1wNJl5ywfU2tm62gDg3vKvcg=";
```
Comment (mweinelt) [inline pkgs/applications/misc/acpic/default.nix:13]:
```suggestion
    hash = "sha256-vQ9VxCNbOmqHIY3e1wq1wNJl5ywfU2tm62gDg3vKvcg=";
```

With an SRI hash you can and should prefer `hash`.

File: pkgs/applications/misc/acpic/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  };
+
+  nativeBuildInputs = [
+    python3Packages.pbr
+  ];
+
+  format = "setuptools";
```
Comment (mweinelt) [inline pkgs/applications/misc/acpic/default.nix:20]: Ah, down here. Please move it just below  version.

**Review by mweinelt — CHANGES_REQUESTED** [review]
Please disable the check phase, since the app does not come with any.

```nix
  # no tests
  doCheck = false;
```

File: pkgs/applications/misc/acpic/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{ lib
+, fetchPypi
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "acpic";
+  version = "1.0.0";
+
```
[the remark this answers is not in the document]
Comment (aacebedo) [reply pkgs/applications/misc/acpic/default.nix]: done

File: pkgs/applications/misc/acpic/default.nix [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  src = fetchPypi {
+    inherit version;
+    inherit pname;
```
[the remark this answers is not in the document]
Comment (aacebedo) [reply pkgs/applications/misc/acpic/default.nix]: done

File: pkgs/applications/misc/acpic/default.nix (hunk 1 above)
Comment (aacebedo) [reply pkgs/applications/misc/acpic/default.nix]: done

File: pkgs/applications/misc/acpic/default.nix (hunk 2 above)
Comment (aacebedo) [reply pkgs/applications/misc/acpic/default.nix]: done

**Pull request merged**